css-four-column-layout/styles.css

```diff
@@ -0,0 +1,48 @@
+body {
+  font-family: Barlow, sans-serif;
+  text-align: center;
+}
+
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+.main-content {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
Instead of making this one class, you should make it two:
```
.row {
  display: flex;
}

.justify-center {
  justify-content: center;
}
```

Then apply both classes where you are currently applying `main-content` in your HTML